identity/app/services/ExactTargetService.scala

```diff
@@ -0,0 +1,44 @@
+package services
+
+import com.exacttarget.fuelsdk.{ETResponse, ETSubscriber}
+import utils.SafeLogging
+import common.ExecutionContexts
+import conf.IdentityConfiguration
+import scala.concurrent.Future
+import scalaz.\/
+
+class ExactTargetService(conf: IdentityConfiguration) extends ExecutionContexts with SafeLogging {
```

Comment:
Does this behaviour not exist in the API already? There are already pages for managing ET mailing lists so this may be redundant duplication?

As mentioned IRL, putting this kind of logic in Frontend isn't great. In particular it means the apps won't be able to use it.